lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/persistence/DBOTeamTest.java

```diff
@@ -83,4 +89,54 @@ public void testRoundTrip() throws DatastoreException, NotFoundException, Unsupp
 		assertEquals(clone, clone2);
 	}
 
+	@TemporaryCode(author = "peter.harvey@sagebase.org", comment = "One time migration of state.  Can be removed after all teams have a state value.")
+	@Test
+	public void testCreateDatabaseObjectFromBackupWithPublicTeam() {
+		Team dto = new Team();
+		dto.setId("123");
+		dto.setEtag(UUID.randomUUID().toString());
+		dto.setCanPublicJoin(true);
+		dto.setCanRequestMembership(false);
+		DBOTeam dbo = new DBOTeam();
+		TeamUtils.copyDtoToDbo(dto, dbo);

```

Comment:
The utility will set the state here, so we are bypassing the actual testing of createDatabaseObjectFromBackup. Instead you can use the TeamUtils.copyToSerializedField to take the team and put it in the properties field of the DBOTeam object.